WcaOnRails/db/structure.sql

```diff
@@ -84,6 +84,7 @@ CREATE TABLE `Competitions` (
   `use_wca_live_for_scoretaking` tinyint(1) NOT NULL DEFAULT '0',
   `allow_registration_without_qualification` tinyint(1) DEFAULT '0',
   `guests_per_registration_limit` int(11) DEFAULT NULL,
+  `events_per_registration_limit` int DEFAULT NULL,
```

Comment:
I'm never quite sure what we expect the migrations to do in this file. Should I have manually changed this to:

```suggestion
  `events_per_registration_limit` int(11) DEFAULT NULL,
```

?

Reply:
Changed to `int(11)` for consistency (squashed into first commit).